packages/client-app/src/views/Request/RequestSection/RequestSection.vue

```diff
@@ -37,6 +37,16 @@ const sections = computed(() => {
 type ActiveSections = (typeof sections.value)[number]
 
 const activeSection = ref<ActiveSections>('All')
+
+watch(activeRequest, (newRequest) => {
+  console.log(newRequest)
```

Comment:
That’s just debug output, isn’t it? Should we remove it?

Reply:
omg yes 🐝